docs/system-overview.rst

```diff
@@ -0,0 +1,84 @@
+Proctoring System Overview
+===========================
+This document outlines the components involved in the edX proctoring system. It should
+serve as a guide to orient developers on what components exist, how they interact, and 
+where to find the source code in our platform.
+
+.. contents::
+
+System Components
+------------------
+
+.. image:: images/components.png
```

Comment:
This seems like a great diagram to put on a slide & talk through, but I don't think it's very self-explanatory. Also several of the features pictured don't exist in RPNow; it's implicitly PT-centric. (So it may make sense to move below where you introduce the prospect of different backends and add some blurb about that.)

An additional comment about the image itself: I feel like "LMS/CMS" & "edx-platform" should probably be reversed in their positions. edx-platform & edx-proctoring are both chunks of code that both run on the LMS, which is an application. So I think the nesting is a little backwards there. Might also make sense to have a different type of box (e.g. a different color) for non-codebase things; the LMS/CMS is not a codebase, as the legend might suggest, nor is the learner's web browser (which we might consider labelling as such, rather than "(web) edX Courseware").

Reply:
I've updated this and attempted to make it a bit less PT centric.  I do like having it up front because I think the visual gives context to each of the components explained below.